Userland/Libraries/LibC/crt0.cpp

```diff
@@ -45,6 +45,8 @@ int _entry(int argc, char** argv, char** env)
 
     s_global_initializers_ran = true;
 
+    program_invocation_short_name = argv[0];
```

Comment:
This is not the same as argv[0]. 

https://man7.org/linux/man-pages/man3/program_invocation_name.3.html

It also belongs in `errno.h`, not in `stdio.h`

>  program_invocation_short_name contains the basename component of name that was used to invoke the calling program.  That is, it is the same value as program_invocation_name, with all text up to and including the final slash (/), if any, removed.